modules/gdscript/tests/scripts/analyzer/errors/bitwise_float_right_operand.gd

```diff
@@ -0,0 +1,3 @@
+func test():
+	# Error here.
+	print(2 << 4.4)
```

Comment:
Should be `>>`.

Reply:
Good catch – I'll fix this in an upcoming PR :slightly_smiling_face: